packages/shared/src/components/tooltips/InteractivePopup.tsx

```diff
@@ -65,9 +71,14 @@ function InteractivePopup({
   position = InteractivePopupPosition.Center,
   closeOutsideClick,
   onClose,
-  closeButtonVariant = ButtonVariant.Secondary,
+  closeButton,
```

Comment:
Just so it won't break the app when the prop is null or undefined.
```suggestion
  closeButton = {},
```

Reply:
I wanted to pass as is to set the defaults in below load, let me know what you think?
(Idea was that you might want to override only 1 for instance) 

```js
const {
    size: buttonSize = ButtonSize.Small,
    variant: buttonVariant = ButtonVariant.Secondary,
    position: buttonPosition = 'right-2 top-2',
  } = closeButton;
```

Reply:
Yes, that snippet would still work as intended, the issue is, JS would throw an error if you destructure the `closeButton` when it is null or undefined.

Since it's an empty object, the `size` prop for example, would be `undefined` as well, in that case, the value would be as what you defined which is `ButtonSize.Small`.